pkg/ccl/backupccl/backup_planning.go

```diff
@@ -40,6 +40,7 @@ import (
 const (
 	backupOptRevisionHistory = "revision_history"
 	backupOptEncPassphrase   = "encryption_passphrase"
+	backupOptWithPrivileges  = "privileges"
```

Comment:
just curious: is it at all feasible to include them as `GRANT` statements in the `SCHEMAS` output instead?